bootstrap.c

```diff
@@ -8,7 +8,7 @@ static const char *get_c_compiler(void) {
     return (cc == NULL) ? "cc" : cc;
 }
 
-static void panic(const char *reason) {
+static __attribute__((noreturn)) void panic(const char *reason) {
```

Comment:
While Windows support in this file is a to-do, I think it would be good to avoid introducing GCC-isms (unconditionally, at least).

Reply:
C11 added _Noreturn specifier, but unfortunately in C23 replaced this with more C++ form.

A good way to do this (to avoid compiler dependent issues):

```c
#if defined (__GNUC__) || defined (__clang__)
# define NORETURN    __attribute__((noreturn))
#else
# define NORETURN
#endif
```

Reply:
I think that is a good idea. I’ll implement this.